packages/nextjs/tsup.config.ts

```diff
@@ -0,0 +1,22 @@
+import { defineConfig } from 'tsup';
+
+import { name, version } from './package.json';
+
+export default defineConfig(overrideOptions => {
+  const isProd = overrideOptions.env?.NODE_ENV === 'production';
+
+  return {
+    entry: ['src/index.ts', 'src/ssr/index.ts', 'src/server/index.ts', 'src/api/index.ts', 'src/client/index.ts'],
```

Comment:
Does this change alter the structure of the `/dist` directory? 

We implement subpath imports (eg `@clerk/nextjs/server` by including the top-level .js and d.ts files (see https://github.com/clerkinc/javascript/blob/00bc852a38c1bc5fc46e9f433c7279c9884108ba/packages/nextjs/package.json#L19-L18)

We need to make sure that these point to the correct /dist files, otherwise the package will break even if publishing succeeds

Reply:
there are some differences in the `dist/`  folder compared to the old one. I will update the tsup config include some subpaths.
@nikosdouvlis  let'ts discuss offline about a way to test if all currently supported exports are covered int he new bundle.